### ISSUE TITLE
Implementation for R

### ISSUE THREAD

**Issue by Timag** [mapnificent/mapnificent#26]
First of all congratulations on a great project! 

I am mostly working with R and the R community would also be interested in this project, i think. "We" also work a lot with leaflet (https://rstudio.github.io/leaflet/) and geo analysis.

I thought about providing a wrapper / R implementation to the community. 
However, it is not straightforward to implement (for me):
1) As far as i understood the code, you make the analysis "tile by tile" which is hard to migrate to the Rstudio leaflet wrapper.
2) Jquery is used, which is only available for R within interactive dashboards (shiny) and not in base R. Would it be a reasonable idea to rewrite the current implementation with underscore.js instead of jquery? (I asked here - how to deal with jquery in base R:https://github.com/jeroen/V8/issues/53)
3) Finally, i am looking at starting from scratch and do an implementation in R (+C). I struggle with loading the .bin files. I found the variable `MAPNIFICENT_PROTO` in your script to be helpful, but i didnt succeed so far.

Long story short: My question would be if you would be open to give guidance / consult me for an (open-source) R implementation?

**Comment by lindes** [comment]
If you haven't already, you may want to look at the code in https://github.com/mapnificent/mapnificent_generator -- that's what generates the .bin files, using go.  Perhaps if nothing else, you could use go to reprocess the file into something different?  Or even just use the python portions to download the CSV files separately, and do native parsing that way.

I'm not super familiar with the javascript side of things, or with R internals, so I'm not saying there aren't other options, just... hoping this might be a useful pointer for things to look at, at least.

**Comment by stefanw** [comment]
I'm working on making the JavaScript parts into individual modules: https://github.com/mapnificent/mapnificent.js

Maybe this gives a bit greater insight on how it works.

**Comment by Timag** [comment]
Thank you for the answers!
@lindes : Yes i saw the generator. I would still focus on `MAPNIFICENT_PROTO` as it seems to be (related to) the protobuf message. I would prefer going with the .bin files as they are ~0,5mb and the raw csvs ~85mb (example for Aachen). Thanks anyway!

@stefanw 
Thanks for sharing -  I will keep an eye on it!
